tests/portability/cmsis_rtos_v1/src/kernel_apis.c

```diff
@@ -48,12 +48,12 @@ void test_kernel_systick(void)
 	diff = (uint32_t)k_cyc_to_ns_floor64(stop_time -
 					 start_time) / NSEC_PER_USEC;
 
-	/* Check that it's within 1%.  On some Zephyr platforms
+	/* Check that it's within 10%.  On some Zephyr platforms
```

Comment:
I don't think increasing tolerance globally is the best approach, if only Nordic's timer has this problem.  Instead a generic tolerance should be defaulted, with an override on platforms that can't meet it for good reasons.

```
#if CONFIG_NRF_RTC_TIMER
#define TOLERANCE_PPC 10
#else
#define TOLERANCE_PPC 1
#endif

...
max = WAIT_TIME_US + (TOLERANCE_PPC * WAIT_TIME_US / 100);
```

Reply:
done